app/config.py

```diff
@@ -772,6 +772,14 @@ class Scratch(Production):
 class Dev(Production):
     FRESH_DESK_ENABLED = env.bool("FRESH_DESK_ENABLED", False)
     NOTIFY_ENVIRONMENT = "dev"
+    # AWS SQS settings override for development
+    BROKER_TRANSPORT_OPTIONS = {
+        "region": AWS_REGION,
+        "polling_interval": 1,  # 1 second
+        "visibility_timeout": 310,
+        "queue_name_prefix": NOTIFICATION_QUEUE_PREFIX,
+        "is_secure": True,  # Use secure connection for SQS in development
```

Comment:
This is the new attribute necessary for this override and that would encrypt the SQS queues as per reported by other Celery users in related issues on this topic. 🤞